styles/colors-copper-day.less

```diff
@@ -82,7 +84,7 @@
 // ---------------------------------------
 @agate-checkbox-text-color: @agate-white;
 @agate-checkbox-bg-color: @agate-accent;
-@agate-checkbox-focus-bg-color: @agate-background-focus;
+@agate-checkbox-focus-bg-color: @agate-focus-bg-color;
```

Comment:
Same here, Is there a reason not to update @agate-picker-item-focus-bg-color like Checkbox?